app/code/Magento/Payment/Helper/Data.php

```diff
@@ -32,11 +32,9 @@ class Data extends \Magento\Framework\App\Helper\AbstractHelper
     protected $_paymentConfig;
 
     /**
-     * Layout
-     *
-     * @var \Magento\Framework\View\LayoutInterface
+     * @var \Magento\Framework\View\LayoutFactory
      */
-    protected $_layout;
+    protected $_layoutFactory;
```

Comment:
Please don't rename exist properties in `@api` class. It's backward incompatible.
Just add another one private property and deprecate current

Reply:
done 👍🏼